ipfshttpclient/client/pin.py

```diff
@@ -55,6 +55,8 @@ def ls(self, type="all", **kwargs):
 
 		Parameters
 		----------
+		path : str
+			The path(s) to object(s) to be listed
```

Comment:
This should be named `cids` like everywhere else (also document the plural parameter, since there is no mandatory first `cid` parameter). Also change the description to something like “Pinned IPFS objects to search for; if any are given, only the requested CIDs will be returned if they are, in fact, pinned – adding CID that is not pinned will result in an error”

Reply:
Renamed, re-wrote description. Btw, it's still 'path' in add,rm,verify.

Reply:
Please unresolve if I didn't get the docs right.